index.html

```diff
@@ -132,5 +132,79 @@ <h1 class="entry-name">Skills</h1>
             <img src="img/skills.svg" alt="" class="back back-frame">
         </div>
     </section>
+    <section class="contact-form">
+        <div class="image">
+            <img src="img/image_geometry_1.svg" alt="">
+        </div>
+        <h1 class="topic begin">Get Started</h1>
+        <p class="text1 begin-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut 
+            labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/xgedqaad" method="post">
+            <ul class="inp-list">
+                <li>
+                    <label for="name">Your name</label>
+                    <input 
+                    type="text" 
+                    name="user_name"
+                    class="inp-field" 
+                    id="name" 
+                    maxlength="30"
+                    required
+                    placeholder="Anastasia">
+                </li>
+                <li>
+                    <label for="email">Your email</label>
+                    <input 
+                    type="email"
+                    pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"
+                    name="user_email"
+                    class="inp-field" 
+                    id="email"                     
+                    maxlength="30"
+                    required>
+                </li>
+                <li>
+                    <textarea 
+                    name="user_message"
+                    class="text-msg" 
+                    id="message" 
+                    cols="30" rows="10" 
+                    placeholder="Enter your message here..."
+                    maxlength="500"
+                    required></textarea>
+                </li>
+            </ul>
+            <button type="button" class="button1">Start collaboration</button>
```

Comment:
- [ ] please make sure your button type is submit, so as to enable the form submission